profit/sur/gaussian_process.py

```diff
@@ -5,6 +5,8 @@
 from .sur import Surrogate
 from profit.util import check_ndim
 
+from sklearn.gaussian_process.kernels import Kernel, Hyperparameter, StationaryKernelMixin, NormalizedKernelMixin
```

Comment:
Why do we add this here? Is there no performance overhead?

Reply:
There is a significant performance overhead, but the previous location of the imports had exactly the same overhead, but less readability. I have not yet found an (easy) way around this import as it is needed in the class declaration.